config.txt

```diff
@@ -60,6 +60,9 @@ hdmi_driver=2
 hdmi_group=2
 hdmi_mode=87
 display_rotate=1
+framebuffer_width=480
+framebuffer_height=256
```

Comment:
I don't think 256 is correct, shouldn't this be 270? Since the display is 1920x1080, and 480 is 1/4 of 1920, 1/4 of 1080 would be 270.

Also, I have been manually adjusting my screen buffer like this:

```
fbset -g 480 270 480 270 16
```

Which looks correct.